extension/src/notification/Widget.tsx

```diff
@@ -0,0 +1,100 @@
+import CloseIcon from "@mui/icons-material/Close";
+import InfoOutlinedIcon from "@mui/icons-material/InfoOutlined";
+import {
+  Card,
+  CardHeader,
+  CardContent,
+  CardActions,
+  IconButton,
+  Checkbox,
+  FormControlLabel,
+} from "@mui/material";
+import { FC, memo } from "react";
+
+import { WidgetContext } from "../shared/context/WidgetContext";
+import { ViewMarkdownViewer } from "../shared/ui-components/common";
+import { PLATEAUVIEW_NOTIFICATION_DOM_ID } from "../shared/ui-components/common/ViewClickAwayListener";
+
+import { useNotificationLogic } from "./hooks/useNotificationLogic";
+
+type NotificationProps = {
+  isEnable?: boolean;
+  content?: string;
+  startTime?: string;
+  finishTime?: string;
+};
+
+type NotificationWidgetProps<NotificationProps> = {
+  inEditor: boolean;
+  widget: {
+    property: {
+      default: NotificationProps;
+    };
+  };
+};
+
+type Props = NotificationWidgetProps<NotificationProps>;
+
+export const Widget: FC<Props> = memo(function WidgetPresenter({ widget }) {
+  const { ready, visible, show, content, handleClose, handleCheckboxChange, doNotShowAgain } =
+    useNotificationLogic();
+
+  if (!visible) return null;
+
```

Comment:
```suggestion
  // Need to import useTheme and useMediaQuery from "@mui/material"
  const theme = useTheme();
  const isMobile = useMediaQuery(theme.breakpoints.down(MOBILE_WIDTH));
```
We can get isMobile here in this way.
In most cases we will use `useMediaQuery(theme.breakpoints.down('mobile'));`, but `mobile` is a customized breakpoint (in customized theme), which is only avaliable inside WidgetContext. For current Notification almost all hooks are out of WidgetContext so i think we need to use the constaint MOBILE_WIDTH directly.

Reply:
I added one more commit to use constant MOBILE_WIDTH.